src/main/java/org/gridsuite/modification/server/service/NetworkModificationService.java

```diff
@@ -233,26 +226,14 @@ public void deleteNetworkModifications(UUID groupUuid, List<UUID> modificationsU
     @Transactional
     public Optional<NetworkModificationResult> moveModifications(UUID destinationGroupUuid, UUID originGroupUuid,
                                                                  UUID beforeModificationUuid, UUID networkUuid, String variantId,
-                                                                 ReportInfos reportInfos, List<UUID> modificationsToMove, boolean canBuildNode) {
+                                                                 ReportInfos reportInfos, List<UUID> modificationsToMove, boolean applyModifications) {
         // update origin/destinations groups to cut and paste all modificationsToMove
-        List<ModificationEntity> movedEntities = networkModificationRepository.moveModifications(destinationGroupUuid, originGroupUuid, modificationsToMove, beforeModificationUuid);
+        List<ModificationInfos> modificationInfos = networkModificationRepository.moveModifications(destinationGroupUuid, originGroupUuid, modificationsToMove, beforeModificationUuid)
+            .stream()
+            .map(networkModificationRepository::getModificationInfos)
+            .toList();
 
-        if (canBuildNode && !movedEntities.isEmpty()) { // TODO remove canBuildNode ?
-            // try to apply the moved modifications only (incremental mode)
-            PreloadingStrategy preloadingStrategy = movedEntities.stream()
-                    .map(e -> ModificationType.valueOf(e.getType()))
-                    .reduce(ModificationType::maxStrategy).map(ModificationType::getStrategy).orElse(PreloadingStrategy.NONE);
-            NetworkInfos networkInfos = getNetworkInfos(networkUuid, variantId, preloadingStrategy);
-            if (networkInfos.isVariantPresent()) {
-                List<ModificationInfos> movedModifications = movedEntities.stream()
-                        .map(networkModificationRepository::getModificationInfos).toList();
-                return Optional.of(modificationApplicator.applyModifications(
-                        movedModifications,
-                        networkInfos,
-                        reportInfos));
-            }
-        }
-        return Optional.empty();
+        return applyModifications ? applyModifications(networkUuid, variantId, reportInfos, modificationInfos) : Optional.empty();
```

Comment:
I remember I made these changes above (lines 240->255) to optimize the move operation specially with Tabular modifications.
Copying the entities are very fast, but converting entities->dto is very slow. So the idea was to delay this conversion at the last time (just before we are sure to apply).

If we want to keep good performances with Tabular modifications (in the case we dont apply), we cannot merge your change above.
I measured: 
- moving a single big Tabular modification (on all lines) at entity level: 250ms
- converting this modification: 4783ms

Imagine if you move several of them (without apply),

Reply:
I change the code to delay this conversion at the last time